### ISSUE TITLE
Some issues about deepspeed

### ISSUE THREAD

**Issue by YuanSun-XJTU** [Wangt-CN/DisCo#11]
Hi, thank you very much for your amazing work. I have successfully run the Gradio Demo using the model you provided. 

However, I encountered the following error during the fine-tuning training phase using fp16 deepspeed: 
`[INFO] [stage_1_and_2.py:1651:step] [deepspeed] OVERFLOW! Rank 0 Skipping step. Attempted loss scale: 1073741824.0, reducing to 536870912.0`. How should I configure `deepspeed.py` to solve this problem?

**Comment by Wangt-CN** [comment]
Hi @YuanSun-XJTU , thanks a lot for your interests. Could you please check if it is an error? Since there is indeed a kind of `OVERFLOW` warning at the start of training, but just last for a while.

**Comment by YuanSun-XJTU** [comment]
Thanks for your response. I have resolved the issue.
1.Make sure that the versions of PyTorch and DeepSpeed are compatible.
2.Set the `initial_scale_power` appropriately for DeepSpeed's fp16.

**Comment by SKBL5694** [comment]
sorry for reopen this issue, I met the same bug with you. Can you tell me you pytorch version and deepspeed version? And how to set the `initial_scale_power`? Thx a lot.

**Comment by YuanSun-XJTU** [comment]
> sorry for reopen this issue, I met the same bug with you. Can you tell me you pytorch version and deepspeed version? And how to set the `initial_scale_power`? Thx a lot.
My GPU is 4090, cuda version is 11.3, pytorch version is 1.12.1, deepspeed version is 0.6.3.
Modify the file [deepspeed.py](https://github.com/Wangt-CN/DisCo/blob/4e870901676eccf83b3e367090e5c945f48194b7/utils/deepspeed.py#L24):
```
config_params['fp16'] = {
       'enabled': True,
       'initial_scale_power': 16,
}

```

**Comment by SKBL5694** [comment]
That't helpful, thank you. : )

**Comment by SKBL5694** [comment]
I encountered like this:
 `[deepspeed] OVERFLOW! Rank 0 Skipping step. Attempted loss scale: 1, reducing to 1` and
`[logging.py:69:log_dist] [Rank 0] step=640, skipped=636, lr=[1.4099400775467043e-07], mom=[[0.9, 0.98]]` 
as you see, it run 640 steps, but skip 636 of that which mean only 4 steps do the real training I guess.
I use the code in readme, can you tell me why? Thx.
/(ㄒoㄒ)/~~
